app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -191,21 +192,31 @@ class SamplesView extends React.Component {
         ? mapSidebarSelectedSampleIds
         : selectedSampleIds;
 
-    const log = () =>
-      logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
-        selectedSampleIds: targetSampleIds.length,
+    const heatmapOptions = [
+      { text: "Taxon Heatmap", value: "/visualizations/heatmap" },
+    ];
+    if (this.props.admin) {
+      heatmapOptions.push({
+        text: "AMR Heatmap",
+        value: "/amr_heatmap",
       });
+    }
     return targetSampleIds.size < 2 ? (
       <HeatmapIcon className={cx(cs.icon, cs.disabled, cs.heatmap)} />
     ) : (
-      <a
-        onClick={log}
-        href={`/visualizations/heatmap?sampleIds=${Array.from(
-          targetSampleIds
-        )}`}
-      >
-        <HeatmapIcon className={cx(cs.icon, cs.heatmap)} />
-      </a>
+      <HeatmapIconDropdown
+        className={cx(cs.icon, cs.heatmapDropdown)}
+        iconClassName={cx(cs.icon, cs.heatmap)}
+        options={heatmapOptions}
+        onClick={heatmapOption => {
+          logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
```

Comment:
Nit: I would use `heatmap-option` instead of `heatmap-icon` since you're picking an option.